packages/file-collections/src/node/getFileDownloadHandler/requestRange.js

```diff
@@ -22,7 +22,7 @@ export default function requestRange(headers, fileSize) {
   }
 
   const defaultRange = {
-    end: fileSize - 1,
+    end: fileSize,
```

Comment:
I'm honestly not sure, but this doesn't seem to match what is done below when there's a range. It looks like we parse into `endByte` and `startByte`, but then we use the original string values in the returned object. Shouldn't it be `end: endByte` and `start: startByte` in the returned object? And if so, `endByte = fileSize - 1` will be different from the value of `end` in this default range.

Reply:
@vanpho93 Would that mean it was correct as it was? I think it would be useful to add several unit tests. Then it should be easy to confirm that the return object correctly represents the input header.

Also I think the main issue here is that the final return isn't returning the correct numbers for start/end. `start` and `end` are strings, so if this was TypeScript it would be complaining. It should be:

```js
return {
    end: endByte,
    len: partSize,
    partial: (partSize < fileSize),
    size: fileSize,
    start: startByte,
    unit
  };
```